services-js/registry-certs/client/birth/components/DisclosureComponent.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from 'react';
+
+import { css } from 'emotion';
+
+import {
+  CLEARDEFAULTSTYLING,
+  OPTIMISTIC_BLUE_DARK,
+} from '@cityofboston/react-fleet';
+
+export default function DisclosureComponent(): JSX.Element {
+  const [detailsElementIsSupported, setDetailsElementIsSupported] = useState<
+    boolean
+  >(false);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const toggleIsOpen = () => {
+    setIsOpen(isOpen => !isOpen);
+  };
+
+  useEffect(
+    () => {
+      // ie11 does not support the <details> element, but we can test for support
+      // by checking if the “open” attribute is present.
+      const testElement = document.createElement('details');
+
+      setDetailsElementIsSupported('open' in testElement);
+    },
+    // Because we only need to test for support once, ensure the effect does
+    // not fire again.
+    [detailsElementIsSupported]
+  );
+
+  if (detailsElementIsSupported) {
+    return (
+      <details className={DETAILS_STYLING}>
+        <summary>{summaryContent()}</summary>
+
+        {detailsContent()}
+      </details>
+    );
+  } else {
+    return (
+      <aside className={FALLBACK_STYLING}>
+        <button
+          type="button"
+          className={CLEARDEFAULTSTYLING.BUTTON}
+          aria-expanded={isOpen}
+          aria-controls="detailsContent"
+          onClick={toggleIsOpen}
+        >
+          {summaryContent()}
+        </button>
+
+        <div id="detailsContent" className="details-content">
+          {detailsContent()}
+        </div>
+      </aside>
+    );
+  }
+}
+
+function summaryContent(): React.ReactChild {
+  return (
+    <>
+      <svg
+        viewBox="0 0 20 20"
+        aria-hidden="true"
+        focusable="false"
+        overflow="visible"
+        className={SVG_STYLING}
+      >
+        <path d="M 2,5 10,20 18,5 Z" />
+      </svg>
+      Are you requesting a certificate for international use that requires an
+      Apostille from the Massachusetts Secretary of State?
+    </>
+  );
+}
+
+function detailsContent(): React.ReactChild {
+  return (
+    <>
+      <p>
+        You need to have a hand signature from the Registry. After you finish
+        your order, please email birth@boston.gov with:
```

Comment:
Do you want to link this to the contact form?

Reply:
Nah - Patty doesn’t want to make this all that easy (so people aren’t requesting signatures for no actual reason). Also, the user is supposed to send the email after the order was placed. There’s got to be a smoother process while satisfying her concerns about excessive requests, but this works for now.